src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.String.cs

```diff
@@ -6,11 +6,32 @@ namespace System.Text.Json.Serialization
 {
     public static partial class JsonSerializer
     {
+        /// <summary>
+        /// Convert the value and return as <see cref="System.String"/>.
+        /// </summary>
+        /// <returns>The converted value.</returns>
+        /// <param name="value">The value to convert.</param>
```

Comment:
I am not sure how much value such a comment provides. If we are keeping it so that we have something show up rather than nothing, then maybe its fine, but I think we should either elaborate or remove it.

@mairaw, do you have any thoughts on this?